interview-questions.md

```diff
@@ -7,52 +7,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props is short for properties in React. It is a way that we can pass data from one component to another. For example, if we have a component instantiated in the main app.js file. We could give it a property of "name" and set it to value. That value could be used in the component file by using props.name .
 
-  Researched answer:
+  <Example 
+    name = "sample"
+  />
+
+  props.name = "sample" in the component the props would be passed onto.
+
+  Researched answer: Props, or properties, are  used to pass data from one component to another. More so, it allows for that data to be dynamic when being passed between components. It allows you to change data in different components without having to rewrite the code in each component.
```

Comment:
An important note is that props are read-only and can only be updated if state is updated.